flixel/system/replay/FlxReplay.hx

```diff
@@ -125,20 +122,22 @@ class FlxReplay
 	
 	/**
 	 * Save the current recording data off to a String object.
-	 * Basically goes through and calls FrameRecord.save() on each frame in the replay.
+	 * Basically goes through and calls FrameRecord.save() on each frame in the replay, and outputs it only if anything happened that frame.
 	 * return	The gameplay recording in simple ASCII format.
 	 */
 	public function save():String
 	{
 		if (frameCount <= 0)
-		{
 			return null;
-		}
-		var output:String = seed+"\n";
+		
+		var output:String = seed + "\n";
+		var curLine:String;
 		var i:Int = 0;
 		while (i < frameCount)
 		{
-			output += _frames[i++].save() + "\n";
+			curLine = _frames[i++].save();
+			if (curLine != null)
+				output += curLine + "\n";
```

Comment:
I don't understand what you mean--you can't record during playback.  Or were your two questions unrelated to each other?  In that case your second question still doesn't make sense to me--what input are you waiting for on a frame where nothing happens?  (Doesn't "nothing happens" mean no input occurs on that frame?)